ruby/ql/lib/codeql/ruby/ApiGraphs.qll

```diff
@@ -306,7 +305,7 @@ module API {
    * shortest length.
    */
   cached
-  private module Impl {
+  module Impl {
```

Comment:
Why this change?

Reply:
I needed that change in the other languages so I could access private predicates.  
I just copy-pasted the change, but it's not needed here, so I'll revert it.